src/module-elasticsuite-core/etc/adminhtml/system.xml

```diff
@@ -180,6 +180,48 @@
                     <comment><![CDATA[Maximum number of popular search terms to display in autocomplete results.]]></comment>
                 </field>
             </group>
+
+            <group id="advanced" translate="label" type="text" sortOrder="99" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Advanced Settings</label>
+                <field id="extension_enabled" translate="label comment" type="select" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Use suggested search terms to fetch results</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <comment><![CDATA[Default: Yes. When enabled, autocompletion box results will be computed from the suggested search terms. This is intended to suggest results when the user is in the process of typing. Eg : When the user is typing "comp", the popular search term "computer" will be suggested, and therefore products or categories matching "computer" will be shown.]]></comment>
+                </field>
+                <field id="preserve_base_query" translate="label comment" type="select" sortOrder="15" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Always use the user raw input for suggestions.</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <depends>
+                        <field id="extension_enabled">1</field>
+                    </depends>
+                    <comment><![CDATA[Default: No. When set to "Yes", this setting will always proceed to fetch suggestions on the query typed by the user in addition to the extended terms. Eg : When the user is typing "comp", the popular search term "computer" will be suggested, but products or categories matching either "comp" or "computer" will be shown.]]></comment>
+                </field>
+                <field id="extension_limited" translate="label comment" type="select" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Limit the amount of suggested search terms used for autocomplete matching</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <depends>
+                        <field id="extension_enabled">1</field>
+                    </depends>
+                    <comment><![CDATA[Default: No. When set to "No", this setting will use the maximum number of suggested popular search terms configured in the dedicated section.]]></comment>
```

Comment:
I would precise "Default: No (legacy behavior)."
and
"When set to "No", all the suggested popular search terms will be used for the products and categories autocomplete."